docs/source/tutorials/text.ipynb

```diff
@@ -107,6 +107,13 @@
     "# Package installation (hidden on docs.cleanlab.ai).\n",
     "# If running on Colab, may want to use GPU (select: Runtime > Change runtime type > Hardware accelerator > GPU)\n",
     "# Package versions we used: tensorflow==2.9.1 scikeras==0.6.1 tensorflow_datasets==4.5.2\n",
+    "\n",
+    "# Supress outputs that may appear if tensorflow happens to be improperly installed: \n",
+    "import os \n",
+    "import logging \n",
+    "os.environ[\"TF_CPP_MIN_LOG_LEVEL\"] = \"3\"  # suppress tensorflow log output \n",
+    "logging.getLogger('tensorflow').setLevel(logging.FATAL) \n",
+    "\n",
```

Comment:
Can we not move this above `dependencies`, which is a variable all tutorial authors must declare and thus should be very close to top of the notebooks.